backend/controllers/login.js

```diff
@@ -0,0 +1,7 @@
+exports.get = (req, res)=>{
+	res.json([{a:1, n:"a"},{a:13, n:"e"},{a:12, n:"c"}]);
```

Comment:
I'm not understanding what you are trying to do  here ?

Reply:
I think it should be related to the login even if it fake data

Reply:
yes, but i used it to test for carers page, i will fix that